tests/data/profile_stability/rhel8/stig_gui.profile

```diff
@@ -490,4 +491,5 @@ cpe_names: !!set {}
 platform: null
 filter_rules: ''
 title: DISA STIG with GUI for Red Hat Enterprise Linux 8
+definition_location: /home/mburket/code/ComplianceAsCode/content/products/rhel8/profiles/stig_gui.profile
```

Comment:
If you want, you can edit this commit by removing this line and so delete the next commit.

Reply:
Done.